src/PortAudio.jl

```diff
@@ -208,7 +213,7 @@ end
 isopen(stream::PortAudioStream) = stream.stream != C_NULL
 
 SampledSignals.samplerate(stream::PortAudioStream) = stream.samplerate
-SampledSignals.blocksize(stream::PortAudioStream) = stream.blocksize
+SampledSignals.blocksize(stream::PortAudioStream) = trunc(Int, stream.samplerate * stream.latency)
```

Comment:
@ssfrr is this line making sense for you?

Reply:
I’m not at my computer right now, but there’s a way to get the exact blocksize out of PortAudio. I think maybe it gets set in the parameters struct when you open the stream.

Reply:
hmm, actually I lied. I just looked through the docs and I must have been mis-remembering.

That said, this way of computing the blocksize isn't reliable either. The device will pick a blocksize based on the requested latency, but there's no guarantee of how close it is.

I think for now we might not be able to support `blocksize`. I'm OK with that.

Reply:
thanks for checking that. I removed SampledSignals.blocksize method and my test program works well on Ubuntu Studio.

Reply:
@ssfrr, I tested also on mac, it works fine. From my perspective this is ready for merge.